components/about/CustomTable.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+    <div class='officer-table'>
```

Comment:
Codacy has a fix for the issue: [Expected to be enclosed by double quotes.](https://app.codacy.com/gh/AIESEC-in-Japan/aiesecjp-web/pullRequest?prid=7667176)
```suggestion
    <div class="officer-table">
```